application/api/authentication/management/commands/user.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework.authtoken.models import Token
+from django.contrib.auth.models import User
+from ....authentication.models import CapacityAuthDosUser
+
+
+class Command(BaseCommand):
+    help = "Create a new user"
+
+    def add_arguments(self, parser):
+        parser.add_argument("username", type=str)
+        parser.add_argument("password", type=str)
+
+    def handle(self, *args, **options):
+        username = options.get("username", None)
+        password = options.get("password", None)
+        try:
+            user = (
+                User.objects.db_manager("default")
+                .filter(username__contains=username)
+                .delete()
+            )
+        except ObjectDoesNotExist:
+            pass
+        finally:
+            user = User.objects.db_manager("default").create_user(
+                username=username, password=password
+            )
+            CapacityAuthDosUser.objects.db_manager("default").create(
+                dos_username="TestUser", dos_user_id=1000000002, user=user
```

Comment:
Maybe something to consider for later, but we could parametise the dos_username so that we can create API users that link to DoS Users with differing services that they are allowed to update. No need to pass the dos_user_id in as we can probably (and should) look that up.

Reply:
Good suggestion. I'm going to try to implement now.